docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -160,6 +163,48 @@ To create a custom decimals implementation, see {custom-decimals}.
 
 NOTE: This information is only used for _display_ purposes: it in no way affects any of the arithmetic of the contract.
 
+[.contract]
+[[IERC20Permit]]
```

Comment:
This shouldn't be part of the `Core` section, but of an `Extensions` section before `Presets`

Reply:
Moved to the `Extensions` section